### PR TITLE
DDT-1036: Use ShipmentLocationService + query test

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: src/main/java/org/dcsa/ebl/model/mappers/TransportDocumentMapper.java [hunk 1]
```diff
@@ -14,7 +14,6 @@ public interface TransportDocumentMapper {
     @Mapping(source = "shippedOnBoardDate", target = "shippedOnboardDate")
     TransportDocumentSummary transportDocumentToTransportDocumentSummary(TransportDocument transportDocument);
 
-
   @Mapping(source = "placeOfIssue", target = "placeOfIssue", ignore = true)
```
Comment (pimg) [inline src/main/java/org/dcsa/ebl/model/mappers/TransportDocumentMapper.java:17]: Indentation seems off

File: src/test/java/ebl/v2/TransportDocumentIT.java [hunk 2]
```diff
@@ -22,18 +23,18 @@ static void configs() throws IOException {
...
-            .body("size()", greaterThanOrEqualTo(0))
-            .body("transportDocumentReference", anyOf(hasItem("9b02401c-b2fb-5009")));
+    given()
+        .contentType("application/json")
+        .get(TRANSPORT_DOCUMENT_SUMMARIES)
+        .then()
+        .assertThat()
+        .statusCode(200)
+        .body("size()", greaterThanOrEqualTo(0))
+        .body("transportDocumentReference", anyOf(hasItem("9b02401c-b2fb-5009")));
```
Comment (pimg) [inline src/test/java/ebl/v2/TransportDocumentIT.java:33]: add schema validation for summaries

File: src/test/java/ebl/v2/TransportDocumentIT.java [hunk 3]
```diff
@@ -58,11 +59,36 @@ void testGetNotFoundTransportDocument(){
...
+
+    // This is to test shipmentLocationRepository.findByTransportDocumentID query
+    // (CI is not configured in Event-Core to test against database)
+    given()
+        .contentType("application/json")
+        .get(TRANSPORT_DOCUMENTS + "/" + "0cc0bef0-a7c8-4c03")
+        .then()
+        .assertThat()
+        .statusCode(HttpStatus.SC_OK)
+        .body("transportDocumentReference", notNullValue())
```
Comment (pimg) [inline src/test/java/ebl/v2/TransportDocumentIT.java:83]: add schema validation

File: src/test/java/ebl/v2/TransportDocumentIT.java (hunk 2 above)
Comment (mp-assco) [reply src/test/java/ebl/v2/TransportDocumentIT.java]: Done

File: src/test/java/ebl/v2/TransportDocumentIT.java (hunk 3 above)
Comment (mp-assco) [reply src/test/java/ebl/v2/TransportDocumentIT.java:83]: Done

File: src/main/java/org/dcsa/ebl/model/mappers/TransportDocumentMapper.java (hunk 1 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/ebl/model/mappers/TransportDocumentMapper.java:18]: Done

**Review by pimg — APPROVED** [review]

**Pull request merged**